docs/index.html

```diff
@@ -392,22 +392,27 @@ <h2>Data Model</h2>
       </p>
 
       <section>
-        <h3 id="Profile">Profile</h3>
+        <h3 id="Collection">Collection</h3>
 
-        <p>A profile is a named bucket for grouping wallet content.</p>
+        <p>A collection is a named bucket for grouping wallet content.</p>
         <p>
-          A profile MAY be used to group all correlatable wallet items together.
+          A collection MAY be used to group all correlatable wallet items together.
         </p>
         <p>
-          A profile MAY be used to manage separate external personal, or
-          organizational identities and their associated data.
+          A collection MAY be used to aggregate personal or
+          organizational identities and their associated data. 
+        </p>
+        <p>
+          Collection membership is not exclusive; e.g., a credential representing
+          college graduation could appear in a "student record" collection as well 
+          as a "professional profile" collection.  
```

Comment:
I should read more carefully... words are often better than letters.
```suggestion
          Collection membership is not exclusive; for instance, a credential
          representing college graduation could appear in a "student record"
          collection as well as a "professional profile" collection.
```